galasa-ui/src/components/TokenRequestModal.tsx

```diff
@@ -9,8 +9,10 @@ import { Button, Modal } from '@carbon/react';
 import { useRef, useState } from 'react';
 import { TextInput } from '@carbon/react';
 import { InlineNotification } from '@carbon/react';
+import { Add } from '@carbon/icons-react';
+
+export default function TokenRequestModal({isDisabled} : {isDisabled : boolean}) {
```

Comment:
Why is `isDisabled` a property? This component lets users create new access tokens, so I don't see this ever needing to be disabled